### PR TITLE
aws-sso-cli: init at 1.9.2

### PR DESCRIPTION
**Pull request by devusb**
###### Description of changes

https://github.com/synfinatic/aws-sso-cli

AWS SSO CLI is a secure replacement for using the [aws configure sso](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-sso.html) wizard; it makes it easy to manage your shell environment variables allowing you to access the AWS API & web console using CLI tools.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by viraptor — COMMENTED** [review]
Result of `nixpkgs-review pr 176447` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>aws-sso-cli</li>
  </ul>
</details>

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -3105,6 +3105,12 @@
     githubId = 17111639;
     name = "Devin Singh";
   };
+  devusb = {
+    email = "mhelton@devusb.us";
+    github = "devusb";
+    githubId = 4951663;
+    name = "Morgan Helton";
+  };
```
Comment (bobby285271) [inline maintainers/maintainer-list.nix:3113]: Please put this in a [separate commit](https://nixos.org/manual/nixpkgs/stable/#var-meta-maintainers) `maintainers: add devusb` so there are exactly 2 commits in this PR.

File: pkgs/tools/admin/aws-sso-cli/default.nix [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    vendorSha256 = "BlSCLvlrKiubMtfFSZ5ppMmL2ZhJcBXxJfeRgMADYB4=";
+
+    postInstall = ''
+      mv $out/bin/cmd $out/bin/aws-sso
+    '';
+
+    meta = with lib; {
+      homepage = "https://github.com/synfinatic/aws-sso-cli";
+      description = "AWS SSO CLI is a secure replacement for using the aws configure sso wizard";
+      license = licenses.gpl3;
```
Comment (bobby285271) [inline pkgs/tools/admin/aws-sso-cli/default.nix:21]: [Not sure](https://github.com/synfinatic/aws-sso-cli/blob/v1.9.2/cmd/main.go#L10), but licenses.gpl3 is deprecated.

```suggestion
      license = licenses.gpl3Plus;
```

File: pkgs/tools/admin/aws-sso-cli/default.nix [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      maintainers = with maintainers; [ devusb ];
+      platforms = platforms.all;
```
Comment (bobby285271) [inline pkgs/tools/admin/aws-sso-cli/default.nix:23]: We don't specify this if the default platform works (#153464)

```suggestion
```

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (devusb) [reply maintainers/maintainer-list.nix:3113]: Done.

File: pkgs/tools/admin/aws-sso-cli/default.nix (hunk 2 above)
Comment (devusb) [reply pkgs/tools/admin/aws-sso-cli/default.nix]: Done.

File: pkgs/tools/admin/aws-sso-cli/default.nix (hunk 3 above)
Comment (devusb) [reply pkgs/tools/admin/aws-sso-cli/default.nix]: Done.

**Comment by devusb** [comment]
@bobby285271 just wanted to check-in -- need anything on my side for this one?

**Comment by bobby285271** [comment]
Ah sorry for keeping you waiting, let's merge this. By the way https://github.com/synfinatic/aws-sso-cli/blob/v1.9.2/Makefile#L40 mentions some of the ldflags, probably you will wish to also set them in some future PRs

**Pull request merged**